src/utils/isHookCreated.js

```diff
@@ -11,7 +11,7 @@ const isHookCreated = async () => {
     if (!fs.existsSync(stdout + HOOK.PATH)) {
       return false
     }
-    return fs.readFileSync(stdout + HOOK.PATH) === HOOK.CONTENTS
+    return fs.readFileSync(stdout + HOOK.PATH, 'utf8') === HOOK.CONTENTS
```

Comment:
I fixed this aswell! So when you rebase it there's no need to change that anymore! 🎉